sw/airborne/autopilot.c

```diff
@@ -87,6 +87,20 @@ static void send_actuators(struct transport_tx *trans, struct link_device *dev)
 }
 #endif
 
+static void send_minimal_com(struct transport_tx *trans, struct link_device *dev)
```

Comment:
Why not to have only a telemetry profile for these data?

I do not see why this is necessary for autopilot.c.

Reply:
This message is reducing the bandwidth merging information from several messages. Indeed, we were using a specific telemetry file with adapted messages frequency. We wanted to avoid saturation on the modems since it was flying multiple aircraft out of line-of-sight, but still keep a good refresh rate (better than minimal at least).
I think there is no harm to keep it, but redefining the complete set of messages is still in the todo list.